automation/nightly/test-nightly-build.sh

```diff
@@ -18,10 +18,13 @@ function dump() {
 
 # Get golang
 $CRI_BIN login --username "$(cat "${QUAY_USER}")" --password-stdin quay.io < "${QUAY_PASSWORD}"
-wget -q https://dl.google.com/go/go1.22.6.linux-amd64.tar.gz
+wget -q https://dl.google.com/go/go1.23.5.linux-amd64.tar.gz
 tar -C /usr/local -xf go*.tar.gz
 export PATH=/usr/local/go/bin:$PATH
 
+# add qemu-user-static
+$CRI_BIN run --rm --privileged docker.io/multiarch/qemu-user-static --reset -p yes
```

Comment:
```suggestion
$CRI_BIN run --rm --privileged docker.io/multiarch/qemu-user-static --reset --persistent yes
```
for readability.

also, is this image available in quay.io as well?
because we can hit the docker.io rate limit

Reply:
This is running once, each night, so I'm not so worry about rate limit. but checking about quai.io

Reply:
but other jobs from kubevirt prow might pull images from docker hub, and they are originated from the same IP.

Reply:
This is (1) not very common; and (b) if it's the same IP, then the image is already pulled locally.